src/main/java/org/jenkinsci/plugins/docker/workflow/client/DockerClient.java

```diff
@@ -135,7 +135,7 @@ public String run(@Nonnull EnvVars launchEnv, @Nonnull String image, @CheckForNu
     }
 
     public List<String> listProcess(@Nonnull EnvVars launchEnv, @Nonnull String containerId) throws IOException, InterruptedException {
-        LaunchResult result = launch(launchEnv, false, "top", containerId);
+        LaunchResult result = launch(launchEnv, false, "top", containerId, "-eo", "pid,comm");
```

Comment:
as we only want to list commands, could be simplified using `-o command` and not use a tokenizer.

Reply:
I already tried this. However, I get the error:
```
$ docker top ${CONTAINER_ID} -eo comm
Error response from daemon: Couldn't find PID field in ps output
```
Maybe there is another simple ps trick to just get the command.